src/colorizer/shaders/colorize_RGBA8U.frag

```diff
@@ -86,12 +88,17 @@ void main() {
   // Data buffer starts at 0, non-background segmentation IDs start at 1
   float featureVal = getFeatureVal(int(id) - 1);
   uint outlierVal = getOutlierVal(int(id) - 1);
+  float normFeatureVal = (featureVal - featureMin) / (featureMax - featureMin);
 
-  if (isinf(featureVal) || outlierVal != 0u) {
-    // outlier
-    gOutputColor = vec4(outlierColor, 1.0);
+  // Mask all values, including outliers, that are outside the range.
+  if (hideOutOfRange && (normFeatureVal < 0.0 || normFeatureVal > 1.0)) {
```

Comment:
I wonder if "hideOutOfRange" should also hide outliers.  This was not specified and we can wait for user feedback on that.  But I imagine that outliers would also be considered "out of range" :)

Reply:
I found that if I just left the outlier color shading as the default, I would get a lot of random outlier flicker, hence the masking that's applied here.